src/auth/guards/roles.guard.ts

```
@@ -0,0 +1,33 @@
+import { UserRole } from '../enums/user-role.enum';
+import { JwtService } from '@nestjs/jwt';
+
+@Injectable()
+export class RolesGuard implements CanActivate {
+  constructor(
+    private reflector: Reflector,
+    private jwtService: JwtService,
+  ) {}
+
+  canActivate(context: ExecutionContext): boolean {
+    const requiredRoles = this.reflector.getAllAndOverride<UserRole[]>(ROLES_KEY, [
```

ESLint

[error] Replace `ROLES_KEY,·[` with `⏎······ROLES_KEY,`

(prettier/prettier)